packages/qwik-city/middleware/request-handler/page-handler.ts

```diff
@@ -130,14 +137,21 @@ function getPrefetchBundleNames(result: RenderResult, routeBundleNames: string[]
   return bundleNames;
 }
 
-export function getQwikCityEnvData(userResponse: UserResponseContext): {
+export function getQwikCityEnvData(
+  requestHeaders: Record<string, string>,
+  userResponse: UserResponseContext,
+  mode: QwikCityMode
+): {
```

Comment:
Would it not be better to switch this to an object argument to allow adding parameters in the future?